site/source/tutorials/003_creating_widgets/index.md

```diff
@@ -235,27 +235,28 @@ If you are following along locally, you should already see the new values. Howev
 At this point, we have a good start to our widget, but it still doesn't look very good. In the next section we'll address that by learning how to use CSS to style our widgets.
 
 ## Styling widgets with CSS modules
-In the first tutorial, we were able to adjust the look of the `Banner` widget by directly manipulating the `styles` property of the generated virtual DOM. While this was effective, it is generally considered better to use external CSS files to establish the look and feel of an application. Dojo leverages [CSS Modules](https://github.com/css-modules/css-modules) to provide all of the flexibility of CSS, plus two additional benefits: type support in TypeScript and localized styling rules to help prevent inadvertent rule collisions.
+We can use CSS files to establish the look and feel of a widget or application.
+Dojo leverages [CSS Modules](https://github.com/css-modules/css-modules) to provide all of the flexibility of CSS, but with the additional benefit of localized styling rules to help prevent inadvertent rule collisions. Dojo 2 also makes use of [typed css modules](https://github.com/Quramy/typed-css-modules), so that we can provide css typing files, so you can target css file in your import statements.
```

Comment:
Dojo 2 also makes use of [typed css modules](https://github.com/Quramy/typed-css-modules), so that we can provide css typing files, so you can target css file in your import statements. -> Dojo 2 also makes use of [typed css modules](https://github.com/Quramy/typed-css-modules), so that we can provide css typing files, enabling you to target css file in your import statements.